refresh.py

```diff
@@ -508,7 +509,7 @@ def get_mteb_data(
             df.drop(columns=["PawsX (fr)"], inplace=True)
 
         # Filter invalid columns
-        cols = [col for col in cols if col in base_columns + datasets]
+        cols = [col for col in cols if col in base_columns + datasets or any([col.split()[0] == d for d in datasets])]
```

Comment:
I am not entirely sure what happens here?

Reply:
Similar problem like in `find_tasks`

Reply:
But I believe that what is in the name of the task in the huggingface split (not language) and I believe it shoudl be there for all datasets unless it is the default subset.

Reply:
I'll try to make without find_task a bit later

Reply:
might also be that I am missing something